app/workers/facilities/pssg_download.rb

```diff
@@ -56,18 +50,20 @@ def round_band(band)
 
     def extract_polygon(drive_time_data)
       rings = drive_time_data&.dig('geometry', 'rings')
+      return if rings.blank?
       geojson = "{\"type\":\"Polygon\",\"coordinates\":#{rings}}"
       RGeo::GeoJSON.decode(geojson)
     end
 
     def download_data
       offset = 0
       loop do
-        response = @drivetime_band_client.get_drivetime_bands(offset, 1)
-        break if response.blank?
+        response = @drivetime_band_client.get_drivetime_bands(offset, 30)
 
+        break if response.blank?
+        
         response.each(&method(:create_and_save_drive_time_data))
-        offset += 1
+        offset += 30
```

Comment:
Just curious, how does an offset of 30 effect the drivetime band data?

Reply:
By requesting a chunk of 30 bands instead of requesting a single band at a time, we significantly speed up our overall download time. The reason we could not download a bigger chunk than 30 is because the PSSG server has a limit of 35 mb per request. We estimate that chunk sizes of 30 will not go over this limit.

Reply:
2 possibly dumb questions, obviously coming from a non-engineer:
1. It may be overkill, but might it be good to have another job that runs separate of this one, to analyze the data and dynamically set the offset so that it grabs the biggest chunk possible without going over 35 mb?
1. What happens if the data changes and 30 becomes too big (i.e. >35 mb)? Would this lower it and try again or just fail?

Reply:
Not dumb questions at all! 

1 is overkill. That would require a lot of extra engineering time and work. The simplest solution would be to choose the chunk size that is small enough to never go over the limit, but big enough that the download doesn't take forever. We could, perhaps, put in retries with an exponential backoff on the offset size or something like that, but I am fairly confident that 30 will work just fine. 

2. If there is a request that is too big and results in an error, then an error will be raised here: https://github.com/department-of-veterans-affairs/vets-api/pull/3479/files#diff-cb075e0b92af9cbd6df776e0cffe7e62R17 
and then that error will be rescued here: 
https://github.com/department-of-veterans-affairs/vets-api/pull/3479/files#diff-6ed8e216d7afc245ca81a1f3f468305eL17

And all that means is, if an error occurs, it will be filed away in Sentry and then that particular piece of code will be skipped. The loop continues and moves onto the next chunk. It will mean that 30 bands do not get downloaded, but we will get an error in sentry informing us of the event and then we can manually go in and decrease the offset size if necessary.